src/backend/access/transam/slru.c

```diff
@@ -421,6 +427,17 @@ SimpleLruReadPage(SlruCtl ctl, int pageno, bool write_ok,
 				/* Now we must recheck state from the top */
 				continue;
 			}
+#ifdef SERVERLESS
```

Comment:
Better to add a hook for LRUPageRead?

Reply:
add SimpleLruReadPage_hook for plugin to read SLRU page.